src/bindings/python/fluxacct/accounting/create_db.py

```diff
@@ -87,18 +87,19 @@ def create_db(
     conn.execute(
         """
             CREATE TABLE IF NOT EXISTS association_table (
-                creation_time bigint(20)                NOT NULL,
-                mod_time      bigint(20)  DEFAULT 0     NOT NULL,
-                deleted       tinyint(4)  DEFAULT 0     NOT NULL,
-                username      tinytext                  NOT NULL,
-                userid        int(11)     DEFAULT 65534 NOT NULL,
-                bank          tinytext                  NOT NULL,
-                default_bank  tinytext                  NOT NULL,
-                shares        int(11)     DEFAULT 1     NOT NULL    ON CONFLICT REPLACE DEFAULT 1,
-                job_usage     real        DEFAULT 0.0   NOT NULL,
-                fairshare     real        DEFAULT 0.5   NOT NULL,
-                max_jobs      int(11)     DEFAULT 5     NOT NULL    ON CONFLICT REPLACE DEFAULT 5,
-                qos           tinytext    DEFAULT ''    NOT NULL    ON CONFLICT REPLACE DEFAULT '',
+                creation_time   bigint(20)                NOT NULL,
+                mod_time        bigint(20)  DEFAULT 0     NOT NULL,
+                deleted         tinyint(4)  DEFAULT 0     NOT NULL,
+                username        tinytext                  NOT NULL,
+                userid          int(11)     DEFAULT 65534 NOT NULL,
+                bank            tinytext                  NOT NULL,
+                default_bank    tinytext                  NOT NULL,
+                shares          int(11)     DEFAULT 1     NOT NULL    ON CONFLICT REPLACE DEFAULT 1,
+                job_usage       real        DEFAULT 0.0   NOT NULL,
+                fairshare       real        DEFAULT 0.5   NOT NULL,
+                max_jobs        int(11)     DEFAULT 5     NOT NULL    ON CONFLICT REPLACE DEFAULT 5,
```

Comment:
Sorry, I know this is still a draft, but I have one suggestion: is `max_jobs` actually `max_running_jobs`? If so, it might be nice to rename now rather than later since `max_jobs` and `max_active_jobs` could be easily confused (at least for me). By renaming `max_jobs` to `max_running_jobs` or `max_runnable_jobs` or whatever, it would make it clear throughout which limit applies to the total number of active jobs vs just runnable jobs.

Reply:
You're one step ahead of me! I was pushing that work off until I got the `max_active_jobs` named throughout the project; changing `max_jobs` to `max_running_jobs` or something is still left to do (and I agree it's probably a necessary change) and will most likely result in a number of changes throughout the project as well, not just in the plugin. Sorry, I probably should have written that as a **TODO** in the PR description.

Reply:
Ok, I apologize for jumping the gun!